tests/test_formatting.py

```diff
@@ -1,24 +1,22 @@
 # -*- coding: utf-8 -*-
 import os
 import sys
-import unittest
 
 
-class FormattingTestCase(unittest.TestCase):
-    def test_formatting_black(self):
-        project_path = os.path.join(os.path.dirname(__file__), "..")
-        gordo_components_path = os.path.join(project_path, "gordo_components")
-        tests_path = os.path.join(project_path, "tests")
-        cmd = [
-            sys.executable,
-            "-m",
-            "black",
-            "--check",
-            "-v",
-            gordo_components_path,
-            tests_path,
-            "--exclude",
-            r".*_version.py",
-        ]
-        exit_code = os.system(" ".join(cmd))
-        self.assertEqual(exit_code, 0)
+def test_formatting_black():
+    project_path = os.path.join(os.path.dirname(__file__), "..")
+    gordo_components_path = os.path.join(project_path, "gordo_components")
+    tests_path = os.path.join(project_path, "tests")
+    cmd = [
+        sys.executable,
+        "-m",
+        "black",
+        "--check",
+        "-v",
+        gordo_components_path,
+        tests_path,
+        "--exclude",
+        r".*_version.py",
+    ]
+    exit_code = os.system(" ".join(cmd))
+    assert exit_code == 0
```

Comment:
I would suggest no doing any conversions of unittests to pytests in this PR, enough changes are going on here.

Reply:
I'll pull this out to another PR :+1: